src/domain/communication/room/room.service.events.ts

```diff
@@ -117,7 +117,7 @@ export class RoomServiceEvents {
         const result = await this.virtualContributorService.askQuestion(
           chatData,
           agentInfo,
-          spaceNameID
+          spaceID
```

Comment:
context space should be loaded using UUID as well - this was missed in 4010

Reply:
Good catch